tests/v2/integration/catalogv2/cluster_repo_test.go

```diff
@@ -817,6 +817,9 @@ func (c *ClusterRepoTestSuite) testClusterRepoRetries(params ClusterRepoParams)
 	})
 	require.NoError(c.T(), err)
 
+	cr, err = c.catalogClient.ClusterRepos().Get(context.TODO(), params.Name, metav1.GetOptions{})
```

Comment:
Why is this necessary?

Reply:
Just removed it. Thanks for the feedback!